bin/dump-routes

```diff
@@ -6,8 +6,9 @@ use Symfony\Component\Console\Application;
 
 require __DIR__.'/../vendor/autoload.php';
 
-$application = new Application();
 $command = new DumpRouterCommand();
+
+$application = new Application();
```

Comment:
I guess we could use https://symfony.com/doc/current/components/console/single_command_tool.html?

Reply:
Ah forget it, I've mixed things up.